utils/cmpcodesize/cmpcodesize/__init__.py

```diff
@@ -1,6 +1,3 @@
-from __future__ import absolute_import
-from .main import main
-
```

Comment:
The error could be fixed with a corresponding change to `utils/cmpcodesize/cmpcodesize.py`:

```diff
diff --git a/utils/cmpcodesize/cmpcodesize.py b/utils/cmpcodesize/cmpcodesize.py
index 0390af7..eb9ecfa 100755
--- a/utils/cmpcodesize/cmpcodesize.py
+++ b/utils/cmpcodesize/cmpcodesize.py
@@ -1,6 +1,6 @@
 #!/usr/bin/env python

-import cmpcodesize
+from cmpcodesize.main import main

 if __name__ == '__main__':
-    cmpcodesize.main()
+    main()
```

Which I wouldn't be opposed to. I wrote it this way because it mirrored the structure of [llvm/utils/lit/lit.py](https://github.com/apple/swift-llvm/blob/3ebdbb2c7e5ce577363994fd0aa0f8409bc68490/utils/lit/lit/__init__.py).